components/Layouts/minors/nav-user.tsx

```
@@ -111,8 +115,15 @@ export function NavUser({
                 posthog.reset();
-                signOut(auth);
-                router.push("/sign-in");
+                if (isFirebaseActive) {
+                  signOut(auth);
+                } else {
+                  localStorage.removeItem("user");
+                  localStorage.removeItem("token");
+                  setUser(null);
+                }
+                router.replace("/sign-in");
+                router.refresh && router.refresh();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)